util/annotations/annotations.go

```diff
@@ -116,6 +116,9 @@ type annoErr struct {
 }
 
 func (e annoErr) Error() string {
+	if e.Query == "" {
+		return fmt.Sprintf("%s", e.Err)
```

Comment:
Can we just do this?

```suggestion
		return e.Err.Error()
```

Reply:
Good point, thanks! Moving this to https://github.com/prometheus/prometheus/pull/13031